features/dev/persoon-adressering-inonderzoek.feature

```diff
@@ -0,0 +1,486 @@
+      | inOnderzoek.datumIngangOnderzoekPersoon.langFormaat | 1 juli 2002 |
+
+  Rule: DatumIngangOnderzoekPartner wordt gezet wanneer voor adressering benodigde velden van de eerste/actuele/ontbonden partner in onderzoek zijn
+
+    Abstract Scenario: adressering benodigde veld(en) van partner is/zijn in onderzoek
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                        | waarde    |
+      | burgerservicenummer         | 555550001 |
+      | geslachtsaanduiding (04.10) | M         |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                           | waarde        |
+      | voornamen (02.10)              | voornamen     |
+      | geslachtsnaam (02.40)          | geslachtsnaam |
+      | aanduiding naamgebruik (61.10) | N             |
+      En de persoon heeft een 'partner' met de volgende gegevens
+      | naam                                     | waarde                    |
+      | aanduiding gegevens in onderzoek (83.10) | <gba in onderzoek waarde> |
+      | datum ingang onderzoek (83.20)           | 20020701                  |
+      En de 'partner' heeft de volgende 'naam' gegevens
+      | naam                  | waarde        |
+      | geslachtsnaam (02.40) | geslachtsnaam |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550001                       |
+      | fields              | adressering.inOnderzoek         |
+      Dan heeft de response een persoon met de volgende 'adressering' gegevens
+      | naam                                                | waarde                                  |
+      | inOnderzoek.aanhef                                  | <aanhef in onderzoek>                   |
+      | inOnderzoek.aanschrijfwijze                         | <aanschrijfwijze in onderzoek>          |
+      | inOnderzoek.gebruikInLopendeTekst                   | <gebruik in lopende tekst in onderzoek> |
+      | inOnderzoek.datumIngangOnderzoekPartner.type        | Datum                                   |
+      | inOnderzoek.datumIngangOnderzoekPartner.datum       | 2002-07-01                              |
+      | inOnderzoek.datumIngangOnderzoekPartner.langFormaat | 1 juli 2002                             |
+
+      Voorbeelden:
+      | gba in onderzoek waarde | aanhef in onderzoek | aanschrijfwijze in onderzoek | gebruik in lopende tekst in onderzoek |
+      | 050000                  | true                | true                         | true                                  |
+      | 050200                  | true                | true                         | true                                  |
+      | 050210                  | true                | true                         | true                                  |
```

Comment:
Als alleen de voornamen van de partner in onderzoek staan heeft dat dan impact op de aanhef, aanschrijfwijze en gebruiktInLopendeTekst van de gevraagde persoon ?

Reply:
klopt. Wordt verwijderd. Dit geldt dan ook voor 050400 en 050410